composer.json

```diff
@@ -6,7 +6,8 @@
     "license" : "BSD-3-Clause",
     "require" : {
         "php"          : ">=5.5",
-        "ext-mbstring" : "*"
+        "ext-mbstring" : "*",
+        "sabre/xml"    : "~0.2"
```

Comment:
This needs an extra digit. 0.2.1 is guaranteed to be compatible, but 0.3 is not.